dll/win32/kernel32/client/thread.c

```diff
@@ -26,29 +26,6 @@ BasepNotifyCsrOfThread(IN HANDLE ThreadHandle,
 
 /* FUNCTIONS ******************************************************************/
 
-static
-LONG BaseThreadExceptionFilter(EXCEPTION_POINTERS * ExceptionInfo)
```

Comment:
That function looks also a little bit like the "BaseExceptionFilter" that is in proc.c

Reply:
But it seems to be a no-op.
I have removed that one as well.